nixos/modules/programs/iotop.nix

```diff
@@ -0,0 +1,18 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.iotop;
+in {
+  options = {
+    programs.iotop.enable = mkEnableOption "iotop + setcap wrapper";
+  };
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ pkgs.iotop ];
```

Comment:
`/run/wrappers/bin` is on `$PATH` already, so I don't think this is needed (or even does anything, as the iotop in `$PATH` will override this one)

Reply:
Oh, guess I was a couple minutes late :)

Reply:
good catch! Fixed in 6a0d2ff7c1d024914a3570b85f1c88df8930b471